plugin-cerberus/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/CerberusPlugin.java

```diff
@@ -183,6 +183,7 @@ protected Optional<Integer> update(Configuration configuration) {
                                 entry.getKey(),
                                 entry.getValue().getUrl(),
                                 entry.getValue().getVersions()))));
+    final List<String> ignoreVersions = configuration.getIgnore();
```

Comment:
Does this need a change to https://github.com/oicr-gsi/shesmu/blob/master/docs/plugin-cerberus.md ?